src/plugins/analyze_RetraceServer.xml.in

```diff
@@ -12,8 +12,8 @@
     <options>
         <import-event-options event="report_Bugzilla"/>
 
-        <option type="text" name="RETRACE_SERVER_URL">
-           <_label>Retrace server URL</_label>
+        <option type="text" name="RETRACE_SERVER_URI">
+           <_label>Retrace server URI</_label>
            <default-value>retrace.fedoraproject.org</default-value>
```

Comment:
This should now be `https://retrace.fedoraproject.org`, as without the scheme, it’s not a valid URI.